tests/clickhouse-test

```diff
@@ -183,15 +184,17 @@ def run_single_test(args, ext, server_logs_level, client_options, case_file, std
         'stderr': stderr_file,
     }
 
-    pattern = '{test} > {stdout} 2> {stderr}'
+    # >> append to stdout and stderr, because there are also output of per test database creation
+    if not args.database:
+        pattern = '{test} >> {stdout} 2>> {stderr}'
```

Comment:
Now stdout never truncated and multiple test runs will duplicate the stdout reference, and stdout is not used in `CREATE DATABASE` query, so it should be overwritten, fixed in https://github.com/ClickHouse/ClickHouse/pull/24411/commits/2129335147dcd6f0092b7488bf2f5d4717c1fcc6